### PR TITLE
feat(components): add code example and link to codepen to info box

### PR DESCRIPTION
**Pull request by chaoran-chen**
### Summary

This adds, for the prevalence-over-time component, the exact code of the component to the info box and adds a link to CodePen to try it out.

### Screenshot

Info box:

image

CodePen:

image

### PR Checklist

- ~~[ ] All necessary documentation has been adapted.~~
- ~~[ ] The implemented feature is covered by an appropriate test.~~

### CODE REVIEW

File: components/src/preact/prevalenceOverTime/prevalence-over-time.tsx [hunk 1]
```diff
@@ -40,32 +40,25 @@ export interface PrevalenceOverTimeInnerProps {
...
-    denominatorFilter,
-    granularity,
-    smoothingWindow,
-    views,
-    confidenceIntervalMethods,
-    lapisDateField,
-    pageSize,
-    yAxisMaxConfig,
-}) => {
+export const PrevalenceOverTimeInner: FunctionComponent<PrevalenceOverTimeProps> = (componentProps) => {
```
Comment (fengelniederhammer) [inline components/src/preact/prevalenceOverTime/prevalence-over-time.tsx:60]: We don't need `PrevalenceOverTimeInnerProps` as a separate type then anymore.

File: components/src/preact/prevalenceOverTime/prevalence-over-time.tsx [hunk 2]
```diff
@@ -268,6 +224,7 @@ const PrevalenceOverTimeInfo: FunctionComponent<PrevalenceOverTimeInfoProps> = (
                         In the bubble chart, the size of the bubble scales with the total number of available sequences
                         from the {granularity}.
                     </InfoParagraph>
+                    <InfoComponentCode componentName='prevalence-over-time' params={componentProps} lapisUrl={lapis} />
```
Comment (fengelniederhammer) [inline components/src/preact/prevalenceOverTime/prevalence-over-time.tsx:227]: This should probably be outside the `views.includes('bubble') && (...)` conditional?

**Review by fengelniederhammer — APPROVED** [review]
Nice!

File: components/src/preact/prevalenceOverTime/prevalence-over-time.tsx (hunk 2 above)
Comment (chaoran-chen) [reply components/src/preact/prevalenceOverTime/prevalence-over-time.tsx]: Oops, yes, it should be outside!

**Review by JonasKellerer — COMMENTED** [review]
Nice idea. I think this can help spreading the word that it is very easy to use and you can only use single components.

File: components/src/preact/components/info.tsx [hunk 3]
```diff
@@ -55,4 +55,90 @@ export const InfoLink: FunctionComponent<{ href: string }> = ({ children, href }
...
+    };
+    return (
+        <>
+            <InfoHeadline2>Build it yourself</InfoHeadline2>
+            <InfoParagraph>
+                You can add this component to your own website using the{' '}
+                <InfoLink href='https://github.com/GenSpectrum/dashboard-components'>
+                    GenSpectrum dashboard components library
+                </InfoLink>{' '}
+                and the following code.
```
[the remark this answers is not in the document]
Comment (chaoran-chen) [reply components/src/preact/components/info.tsx:80]: I like your suggested phrasing! I also agree that a more human readable format would be nice but for now, I think that showing the code is a good start.

**Pull request merged**